compiler/integration-tests/test/integration/browser/compile_prove_verify.test.ts

```diff
@@ -132,14 +132,15 @@ test_cases.forEach((testInfo) => {
 
             const acirComposer = await api.acirNewAcirComposer(CIRCUIT_SIZE);
 
+            // This took ~6.5 minutes!
             const proof = await api.acirCreateProof(
```

Comment:
6.5 minutes is quite a long time, perhaps this is running in single threaded mode

Reply:
In the aztec-packages repo which also has tests that mirror these, it takes on the order of seconds, so definitely suspicious

Reply:
This is orthogonal and was already there so not blocking